lib/ansible/cli/__init__.py

```diff
@@ -419,8 +423,8 @@ def base_parser(usage="", output_opts=False, runas_opts=False, meta_opts=False,
 
         if module_opts:
             parser.add_option('-M', '--module-path', dest='module_path', default=None,
-                              help="prepend path(s) to module library (default=%s)" % C.DEFAULT_MODULE_PATH,
-                              action="callback", callback=CLI.unfrack_path, type='str')
+                              help="prepend coloon separated path(s) to module library (default=%s)" % C.DEFAULT_MODULE_PATH,
```

Comment:
since force-push ate my previous commit to fix this:
s/coloon separated path(s)/path(s) (colon-separated)/